internal/flink/command_savepoint_describe.go

```diff
@@ -0,0 +1,76 @@
+package flink
+
+import (
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/v4/pkg/cmd"
+	"github.com/confluentinc/cli/v4/pkg/output"
+)
+
+func (c *command) newSavepointDescribeCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:         "describe <name>",
+		Short:       "Describe Flink savepoint in Confluent Platform.",
+		Annotations: map[string]string{pcmd.RunRequirement: pcmd.RequireCloudLogout},
+		RunE:        c.savepointDescribe,
+	}
+
+	cmd.Flags().String("environment", "", "Name of the Flink environment.")
+	cmd.Flags().String("application", "", "The name of the Flink application to create the savepoint for.")
+	cmd.Flags().String("statement", "", "The name of the Flink statement to create the savepoint for.")
```

Comment:
Same comment here about the wording of the flag since this isn't the create command.